js/letters.js

```diff
@@ -16,10 +16,17 @@ var counter = {
 }
 
 function countLetters(counter, sample_text){
-  // FIX ME
-}
+  let lowerCase = sample_text.toLowerCase().replace(/[. , ' ']/g, '').split('').toString();
+  let firstChar = lowerCase.charAt(0);
+  let rest = lowerCase.slice(1);
+  if(sample_text.length === 0){
```

Comment:
Typically the base case would be the first thing in this function because you don't want to do any work with sample_text if you don't have too.